tpmutil/tbs/tbs_windows_test.go

```diff
@@ -111,3 +111,78 @@ func TestSubmitCommandLongResponse(t *testing.T) {
 		t.Fatalf("Got response of %v, expected %v", rawResponse, expectedGetRandomRawResponse)
 	}
 }
+
+// Get Storage owner authorization delegation blob
+func TestGetStorageOwnerAuth(t *testing.T) {
+	ctx := getContext(t)
+	defer ctx.Close()
+
+	authBufferLength, err := ctx.GetOwnerAuth(Storage20Authorization, nil)
+	if err == ErrOwnerauthNotFound || err == ErrInternalError {
```

Comment:
Are you sure that this check will only skip the test if the TBS call failed due to lack of admin rights?